src/Aura/Build.hs

```diff
@@ -76,23 +80,24 @@ build' :: [Buildable] -> Aura ([FilePath], [Buildable])
 build' []     = failure "build' : You should never see this."
 build' (p:ps) = ask >>= \ss -> do
   let user = buildUserOf ss
-  curr   <- liftIO pwd
+      quiet = suppressMakepkg ss
+  curr   <- liftShelly pwd
   getBuildScripts p user curr
   overwritePkgbuild p
-  pNames <- join (makepkg <*> pure user)
+  pNames <- liftShelly (makepkg quiet user []) >>= wrap
```

Comment:
Should not use the empty list here, instead actually pass arguments to makepkg. it's there for type checking purposes right now.